sycl/include/sycl/ext/oneapi/matrix/matrix-jit.hpp

```diff
@@ -11,6 +11,7 @@
 #include <CL/__spirv/spirv_ops.hpp>
 #include <CL/sycl/detail/defines_elementary.hpp>
 #include <CL/sycl/feature_test.hpp>
+#include <sycl/ext/intel/experimental/bfloat16.hpp>
```

Comment:
Buildfail will happen since

class [[sycl_detail::uses_aspects(ext_intel_bf16_conversion)]] bfloat16 

and sycl_detail::uses_aspects is unsupported code in g++ which is used for intel/llvm's build